app/assets/javascripts/html5_webshim.js

```diff
@@ -0,0 +1,15 @@
+$(document).on('ready page:load', function(){
+    window.webshim.setOptions('forms-ext', {
+    lazyCustomMessages: true,
+    replaceUI: true,
+    waitReady: true,
+    type: 'date',
+    date: {
+      startView: 2,
+      openOnFocus: true
+    }
+  });
+
+  window.webshim.polyfill('forms-ext');
+  window.onunload = function(){};
```

Comment:
Hmm, that seems fishy to me.  Is there any way I can get you to give me the steps to replicate that issue (or show me)?  From what I understand, binding to the `ready` and `page:load` events should handle that case.